Samples/Presence/PlayerWatcher/Program.cs

```diff
@@ -47,28 +58,27 @@ static void Main(string[] args)
 
                 // Subscribe for updates
                 var watcher = new GameObserver();
-                game.SubscribeForGameUpdates(GrainClient.GrainFactory.CreateObjectReference<IGameObserver>(watcher).Result).Wait();
+                await game.SubscribeForGameUpdates(
+                    await client.CreateObjectReference<IGameObserver>(watcher));
 
-                // Block main thread so that the process doesn't exit. Updates arrive on thread pool threads.
                 Console.WriteLine("Subscribed successfully. Press <Enter> to stop.");
-                Console.ReadLine();
             }
             catch (Exception exc)
             {
                 Console.WriteLine("Unexpected Error: {0}", exc.GetBaseException());
             }
         }
+    }
```

Comment:
it will be an good example to user which reminds them to use `client.Close() ` and `client.Dispose()` to clear the resources client uses before their client app exit.